contribs/msi/make_x64msi.sh

```diff
@@ -20,6 +22,13 @@ version=$(grep "product.*version"  org.mwc.debrief.product/debriefng.product  |
 sed -i "s/versionReplacement/$version/g" ${WORKDIR}Debrief64.wxs
 echo "Done."
 
+echo "Downloading JRE"
+wget -O jre.zip ${JRE_URL}
+unzip jre.zip -d ${SOURCEDIR}
+mv ${SOURCEDIR}jdk* ${SOURCEDIR}jre
```

Comment:
Hmm, @saulhidalgoaular  - this looks surprising.  Why is the folder called `jdk`?

Reply:
Good evening. Yes, I wondered the same. They have named it as `jdk8u212-b03-jre` . but it is actually a `jre`.